packages/generator-openmrs/templates/src/openmrs-backend-dependencies.ts

```diff
@@ -0,0 +1,3 @@
+export const backendDependencies = {
+  'webservices.rest': '2.24.0',
```

Comment:
You might consider mining some ideas from [my original PR](https://github.com/openmrs/openmrs-frontend-generators/pull/1/files).

```suggestion
  /* Put the OpenMRS modules that this ESM depends on here, e.g. */
  /* "webservices.rest": "2.24.0" */
```

Reply:
Actually, now that I think of it -- I know this would be a departure from existing patterns, but what if we just kept backendDependencies in `index.ts`? Do you think people might forget about it less?

Reply:
I think its a good idea to have everything in index.ts that describes how this MFE will work. Of course, it would be bundled to this later anyway, but having it already in source in a single file would certainly help,